assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+   - Set up a process for maintaining/pruning stale issues
+   - New contributors/your first contribution
+   - Letting new users know where to get help
+
+- Complete the governance work started here: https://github.com/notaryproject/notaryproject/issues/55 https://github.com/notaryproject/notaryproject/pull/78, and add it to a governance page (or section) on the website.
+
+
+## Final Recommendations
+
+Is it possible plan documentation along side the V2 development? Will the documentation be able to get its structure from the specifications being developed?
+
+### Website
+
+- [x] Create separate documentation repository: https://github.com/notaryproject/notaryproject.dev
+- [ ] Setup a [Hugo website](https://gohugo.io) with a [Docsy theme](https://www.docsy.dev/)
+- [ ] Setup Netlify for hosting/CI
+- [ ] Develop a [maintainers list](https://github.com/notaryproject/notaryproject.dev/blob/main/.github/settings.yml) for the documentation and its repository (this has been started https://github.com/notaryproject/notaryproject/issues/77)
+- [ ] Develop a maintenance plan for the documentation and its repository 
+- [ ] Follow the [CNCF website guidelines checklist](https://github.com/cncf/techdocs/blob/main/howto/website-guidelines-checklist.md) for other required elements
+
+_Note_: [Notary Project branding artwork](https://github.com/cncf/artwork/tree/master/projects/notary) exists. Other branding elements, like colour schemes and whatnot, will need to be developed.
+
+### Information Architecture
+
+The content to be written could have the following structure (as a start - modifications will need to be made as development continues):
+
+**Site structure**
+
+- Landing page
+- Community
+   - Governance
+   - Community (details)
+   - Contributing
+      - Where to start
+      - How we work
+      - Filing and working on issues for the project
+      - Where the community is (contacts, meetings)
+      - Contributing to docs
+      - Where to ask for more help contributing
+      - Next steps
+- About
+- Blog (Get the word out about the project and how it's progressing)
+- Docs
+   - This section to be planned with the Notary Maintainers team
+   - Getting Started
+      - This section to be planned with the Notary Maintainers team
+
+**Topnav**
+
+- Docs/Specification (Specification initially, then Docs as development progresses)
+- Version 1 Docs (https://github.com/theupdateframework/notary/tree/master/docs initially, then /docs/v1/(latest?))
+- Community
+- Getting started
+- Specification (https://github.com/notaryproject/notaryproject initially)
+
+
+### Moving and consolidating existing documentation
+
+- We recommend that the V1 Notary Project that is currently housed in the [The Update Framework](https://github.com/theupdateframework/notary/) GitHub organization be moved into the [Notary Project](https://github.com/notaryproject) GitHub organization.
```

Comment:
```suggestion
- We recommend moving the v1 Notary Project project from the [The Update Framework](https://github.com/theupdateframework/notary/) GitHub organization to the [Notary Project](https://github.com/notaryproject) GitHub organization.
```

A  good way to proofread your own work is to look through it and see if there are words you can remove or combine into one wor.d For example, 

"that is currently housed" -> "that is in", "in" both work. 

In this case, rearranging the sentence to use active voice (moved -> moving) removes lots of the extra words.